docs/install/install-cave-beta-v20.md

```diff
@@ -142,36 +132,18 @@ Once inside the VM, to run CAVE either:
 
```

Comment:
Above in the "download and Installtation Instructions" for Linux Virtual Machine

Update version to 0.3 in 3 locations